content/en/docs/profiles.md

```diff
@@ -0,0 +1,132 @@
+
+# Our Profiles
+
+Below are descriptions of each profile, including what effects they have on both the validation process and the contents of the issued certificate. Note that not all profiles are available in all environments: some may be available only in Staging or only in Production, and some may be (temporarily) locked behind an allowlist so we can roll them out slowly. The list of profiles advertised in the ACME Server's `directory` endpoint is the canonical list.
+
+You can find detailed definitions of the properties discussed in each profile at the bottom of this page.
+
+<div class="boxed">
+
+## classic
+
+The classic profile is the default profile selected for all orders which do not request a specific profile. The validation process and resulting certificate are the same as you're used to from the last several years of Let's Encrypt operation. We recommend using this profile for subscribers who are happy to let others try new things first.
+
+| Property | Value |
+| -------- | ----- |
+| [Pending Authorization Lifetime](#pending-authorization-lifetime) | 7 days |
+| [Authorization Reuse Period](#authorization-reuse-period) | 30 days |
+| [Order Lifetime](#order-lifetime) | 7 days |
+| [Certificate Common Name](#certificate-common-name) | Yes<sup>*</sup> |
+| [Key Encipherment KU](#key-encipherment-key-usage) | Yes<sup>†</sup> |
+| [TLS Client Auth EKU](#tls-client-authentication-extended-key-usage) | Yes |
+| [Subject Key ID](#subject-key-identifier-extension) | Yes |
+| [Validity Period](#validity-period) | 90 days |
+
+<sup>\*</sup>: If the CSR submitted at finalize time requests a specific Common Name, that request is honored. If the the CSR does not request a specific Common Name, the first Subject Alternative Name requested will be promoted into the Subject Common Name. If either the requested name or the to-be-promoted name is too long to fit in the Common Name field (64+ characters), the Common Name will be left empty.
+
+<sup>†</sup>: Only included for certificates with RSA public keys.
+
+</div>
+<div class="boxed">
+
+## tlsserver
+
+The tlsserver profile is a new profile which updates several of these validation and certificate properties in order to reflect the latest recommendations from the CA/Browser Forum Baseline Requirements, as well as general trends within the WebPKI community. We recommend selecting this profile for subscribers who want smaller certificates and who fully embrace automation.
```

Comment:
Is it worth saying something like:
> general trends within the WebPKI community, **as of 2025**.

Being that there will be new recommendations that we might not necessarily roll into the tlsserver profile immediately. Or is it the intent that tlsserver will be continuously updated?

Reply:
We do intend to evolve the tlsserver profile as requirements and best practices change.